src/components/colorscale/color_attributes.js

```diff
@@ -64,14 +69,16 @@ module.exports = function makeColorScaleAttributes(context) {
         autocolorscale: extendDeep({}, colorScaleAttributes.autocolorscale, {
             description: [
                 'Has an effect only if `', context, '.color` is set to a numerical array.',
-                ' Determines whether or not the colorscale is picked using',
-                ' values inside `', context, '.color`.'
+                ' Determines whether the colorscale is the default palette (`autocolorscale: true`)',
+                ' or the palette determined by `', context, '.colorscale`.'
             ].join('')
         }),
         reversescale: extendDeep({}, colorScaleAttributes.reversescale, {
             description: [
                 'Has an effect only if `', context, '.color` is set to a numerical array.',
-                ' Reverses the colorscale.'
+                ' Reverses the color mapping if true (`cmin` or lowest `color` value will',
+                ' correspond to the last color and `cmax` or highest `color` value will',
+                ' correspond to the first color).'
```

Comment:
I like the phrase color mapping > colorscale.
I like this additional info but Idk if 'lowest `color` // highest `color`' are necessary (min/max are pretty universally clear imo). I would probably go with:
' Reverses the color mapping if true (`cmin` will correspond to the last color in the array and `cmax` will correspond to the first color).'